tools/server_tests

```diff
@@ -30,9 +30,26 @@ pushd "$(dirname $0)" >/dev/null && source common.sh && popd >/dev/null
 # directory.
 cd "$(dirname $0)/.."
 
+# Parse arguments and set flags
+SKIP_UPDATE_TRANSLATIONS='false'
```

Comment:
Can you use boolean `true` and `false` instead of string `'true'` and `'false'`?
https://stackoverflow.com/a/2953673

Reply:
Done.